### ISSUE TITLE
docs are outdated

### ISSUE THREAD

**Issue by mcgingras** [tildeclub/tilde.club#91]
The 'how to create your own tilde.club' docs seem to be a bit outdated wrt current aws UI. I'm a complete novice to all of this (especially aws) so it is a bit hard to follow.

perhaps the docs could use updating? Seems last update was 5 years ago! I love this project.

**Comment by mbuchholz81** [comment]
Hopefully someone has some experience with current AWS. Or have the doc adapted to some other provider.

**Comment by mcgingras** [comment]
hey -- I was able to get a small tilde.club clone running last night. The docs were helpful for pointing me in the right direction, but as I mentioned, as a complete noob following the instructions step by step was a bit hard because I think AWS changed their UI in the past 5 years.

If you would like, I'd be happy to update the docs for that tutorial with how I got mine set up in AWS and configured with a custom domain.

we are over at old-dogs.club if anybody is interested :)

**Comment by coreyreichle** [comment]
Should docs be AWS specific?  Or, more generic?

Because once you get past the "How to provision a machine in provider X", it should be pretty straight forward, and there are many cloud providers other than AWS these days, contrasted with when tilde.club was founded, and AWS was almost the only reasonable game in town.

**Comment by benharri** [comment]
At least considering the tildes that I'm aware of, none of them are running on AWS anymore. There are much more affordable options for hosting a long-term box nowadays.

**Comment by mcgingras** [comment]
Fair enough, having pretty close to zero experience with any of this stuff, AWS was what I decided to go with since it seems pretty popular. It was also free to set up a micro ec2 instance, I imagine if a certain "club" was to scale you would need to upgrade, but for someone like myself just poking around I had no problems with AWS.

Agreed though, perhaps having guides for different providers of choice would make sense. As mentioned, once you have the machine set and can ssh in, everything else to configure is the same.

**Comment by michaelcoyote** [comment]
Thinking about @benharri 's point that there's better options, it might be good that a document capture people's experiences with setting up and running different tildes on different providers.  The actual setup may change over time per provider so better to document general knowledge like "these are the hard limits that will make your tilde run like crap, so keep these things in mind when sizing your vm", and "if x is happening, consider increasing x"

**Comment by benharri** [comment]
Would love to start a guide to add to our [wiki](https://tilde.club/wiki/), which you can send a PR for with new articles etc on the [tildeclub/site](https://github.com/tildeclub/site/tree/master/wiki) repo.

**Comment by benharri** [comment]
Re: docs on creating your own tilde: there was a submission to the [tildeverse zine](https://zine.tildeverse.org) about [starting your own tilde](https://zine.tildeverse.org/issue-1.html#so-you-wanna-make-a-tilde). Not sure if the details there are specific enough but it covers the motivation behind it.
